prow/plank/README.md

```diff
@@ -0,0 +1,41 @@
+# Plank
+
+Plank is the controller that manages the job execution and lifecycle for jobs running in k8s.
+
+### Usage
+```bash
+$ bazel run //prow/cmd/plank -- --help
+```
+
+### Configuration
+The following job template is based on the format for `spyglass`.
+
+```yaml
+# config.yaml
+
+plank:
+  allow_cancellations: true
+  job_url_prefix: 'https://<domain>/view/gcs/<bucket-name>'
+  job_url_template: 'https://<domain>/view/gcs/<bucket-name>/pr-logs/pull/{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}'
```

Comment:
potentially we should omit this

Reply:
@cjwagner WDYT -- new deployments use pod-utils and don't need it?

Reply:
which link does this generate for btw? `presubmits`? `postsubmits`? the link on the eye icon or the text of the job name?

Reply:
If it is unneeded then it should be omitted.
This is used to generate links for all job types and is used for the link in the text of the job name (the eye always goes to spyglass AFAIK).

Reply:
btw, i've spotted that links for the job name and spyglass doesn't show up when i omit the `job_url_prefix`

Reply:
When you omit the prefix, or the template?

Reply:
`job_url_prefix`

Reply:
Yeah so that's expected -- without it, `plank` does not know how to generate the URLs. Can we remove `job_url_template` from this doc, then?

Reply:
doesn't that also result in job name not having a link to the job log? or is the eye icon for spyglass is enough?

Reply:
It will only not generate a URL for jobs that do not `decorate: true` -- which is how we're expecting everyone to use Prow. @cjwagner @BenTheElder for future deployments of Prow, what do you think we should document here?

Reply:
Maybe just add a comment to the YAML explaining that `job_url_prefix` is used for pod util jobs and `job_url_template` is used for non-pod util jobs?

Reply:
updated